src/Library/demos/Drag and Drop/main.js

```diff
@@ -0,0 +1,88 @@
+import Gtk from "gi://Gtk";
+import Adw from "gi://Adw";
+import Gdk from "gi://Gdk";
+import GObject from "gi://GObject";
+
+const list = workbench.builder.get_object("list");
+const drop_target = Gtk.DropTarget.new(Gtk.ListBoxRow, Gdk.DragAction.MOVE);
+
+list.add_controller(drop_target);
+
+// Iterate over ListBox children
+for (const row of list) {
+  let _drag_x;
+  let _drag_y;
+
+  const drop_controller = new Gtk.DropControllerMotion();
+
+  const drag_source = new Gtk.DragSource({
+    actions: Gdk.DragAction.MOVE,
+  });
+
+  row.add_controller(drag_source);
+  row.add_controller(drop_controller);
+
+  // Drag handling
+  drag_source.connect("prepare", (source, _x, _y) => {
```

Comment:
Update the callbacks, so that arguments that are used aren't prefixed with `_`, like `x` and `y` here. Arguments that aren't used like `source` should be `_source`.